scripts/tcl_commands/synthesis.tcl

```diff
@@ -74,6 +74,11 @@ proc run_yosys {args} {
 }
 
 proc run_synth_exploration {args} {
+    if { $::env(SYNTH_NO_FLAT) } {
+        puts_err "Cannot run synthesis exploration with SYNTH_NO_FLAT."
```

Comment:
Why not?

Reply:
abc no longer outputs "totals", which causes the analysis script to go haywire.

This really isn't worth fixing imo.